src/Snippets/Docs/Migration.Policies.cs

```diff
@@ -106,14 +106,14 @@ public static async Task Strategies()
         // Synchronous execution
         pipelineT.Execute(() =>
         {
-            // your code here
+            // Your code here
```

Comment:
I am always undecided whether we should end these comments with period or not. Maybe we can agree on some pattern an apply it everywhere?

Reply:
I don't mind either way. Typically I do `.` for multiple lines/sentences and omit for for one-liners.